packages/extension-chrome/package.json

```
@@ -35,6 +35,7 @@
   },
   "devDependencies": {
     "@pmmmwh/react-refresh-webpack-plugin": "0.5.10",
+    "@types/rimraf": "^3.0.2",
```

```suggestion
    "@types/rimraf": "3.0.2",
```

We'll use Renovate to upgrade dependencies, so we can make the dependency versions to be fixed